api-controllers/ServiceFabrikApiController.js

```diff
@@ -282,25 +282,47 @@ class ServiceFabrikApiController extends FabrikBaseController {
               })
               .then(() => {
                 lockedDeployment = true;
-                return eventmesh.apiServerClient.createOperation({
-                  resourceId: req.params.instance_id,
-                  operationName: CONST.OPERATION_TYPE.BACKUP,
-                  operationType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
-                  operationId: backupGuid,
-                  value: backupOptions
+                return eventmesh.apiServerClient.createResource({
+                  resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.BACKUP,
+                  resourceType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
```

Comment:
Can same plans have different types ? Or they are at plan level? Because if they are at same plan then there is no distinguishing factor from the input payload.

Reply:
For a certain plan one has to declare which backup type it supports. So it’s one to one mapped.

Reply:
ok.